### DISCUSSION TITLE
Develop question and answer system (quiz).

### DISCUSSION THREAD

**Discussion by wallacebenetti** [community/community#131353] [New to GitHub]
Hello everyone, I'm new to github and I don't know much about coding and development. In fact, I am an enthusiast in this field, and I sell digital products always looking for innovation. I recently came across a sales page that is getting a lot of traffic: https://quiz.wallpilates.com/

There are companies that provide the service of creating this type of question and answer "funnel", and these companies are based on github. Here in Brazil, the most famous is https://inlead.online 

I would like to create my own internal system, where I wouldn't have to pay a monthly fee to have the same question and answer tool. Analyzing the source code of the question and answer quiz block assembly page, I found several github domains, so I imagine this system was developed here. 
Can anyone provide this development service?

**Comment by ghostinhershell** [comment]
Hi @wallacebenetti, we’re glad you’re here 😀

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
